cdap-examples/UserProfiles/README.md

```diff
@@ -0,0 +1,72 @@
+# User Profiles Example
+
+Example application that manages user profiles in a Table. The fields of a user
+profile are updated in different ways:
+
+- Attributes like name and email address are changed through a REST
+  call when the user updates his profile;
+- The time of the last login is updated by a sign-on service every
+  time the user logs in, also through a REST call;
+- The time of the last activity is updated by a flow that processes
+  events whenever it encounters an event from that user.
+
+Features introduced: Column-Level Conflict Detection.
+
+This application uses a table with conflict detection at the column level.
+A conflict occurs if two transactions (for example, a flowlet's process method
+and a service handler) that overlap in time modify the same data in a table.
+Such a conflict is detected at the time that the transactions are committed,
+and the transaction that attempts to commit last is rolled back.
+
+- By default, the granularity of the conflict detection is at the row level.
+  That means, it is sufficient for two overlapping transactions to write to
+  the same row of a table to cause a conflict, even if they write different
+  columns.
```

Comment:
By default, the granularity of the conflict detection is at the row level.
That means it is sufficient for two overlapping transactions writing to
the same row of a table to cause a conflict, even if they write to different
columns.